### DISCUSSION TITLE
Suggestion: Running fix

### DISCUSSION THREAD

**Discussion by yugo76** [Stellarium/stellarium#1597] [Ideas]
Hello. 

 Stellarium could accept a true heading and a speed (either knots or mph/kph) and update the view as we simulate a boat in motion. Even having the proper background (boat) would be nice. This is to practice a "Running fix" in celestial navigation. Oh, and also the option of hiding the current coordinates in the main panel.

**Comment by alex-w** [comment]
It's maybe not an easy to implementation, because some specific features (like movement of observer) of celestial navigation is not in primary focus of any planetarium.

**Comment by axd1967** [comment]
This can probably be achieved by a user script that takes a heading/speed as input and updates the position continuously.

Problems with this approach:
- the running script might hijack further user interactions
- there could be a need to provide a small UI library that facilitates accepting user input (a dialog box)

A plugin is probably the solution for this.

**Comment by gzotti** [comment]
You can simply write a remote control in the language of your choice that does this via the HTTP RemoteControl interface. And create your own "landscape" boat horizon. This can even be rotated to adjust for heading (not sure about a programmable slot for that, we could add this one if necessary). We will not do that special application for you.
